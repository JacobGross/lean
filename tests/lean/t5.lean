constant N : Type.{1}
constant f : N â†’ N
constant a : N
definition g (a : N) : N := f a
check g
namespace foo
  definition h : N := f a
  check h
  private definition q : N := f a
  check q
end foo
check foo.h
check q -- Error q is now hidden
