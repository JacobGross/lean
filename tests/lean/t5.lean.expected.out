g : N â†’ N
h : N
q : N
foo.h : N
t5.lean:13:6: error: unknown identifier 'q'
